content/es/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,781 @@
+Si el sondeo falla, el kubelet mata el contenedor,
+y el contenedor está sujeto a su [política de reinicio](#restart-policy).
+Si un contenedor no tiene un sondeo de liveness, el estado por defecto
+es `Success`.
+
+`readinessProbe`
+: Indica si un contenedor está preparado para responder a peticiones.
+Si el sondeo falla,
+el controlador de endpoints elimina las direcciones IP del Pod de los endpoints
+de todos los Services que coinciden con el Pod.
+El estado por defecto de readiness antes del retraso inicial es `Failure`.
+Si un contenedor no tiene un sondeo de readiness, el estado por defecto
+es `Success`.
+
+`startupProbe`
+: Indica si la aplicación dentro del contenedor ha iniciado. El resto de los
+sondeos están deshabilitados si un sondeo de inicio se proporciona, hasta que se
+complete. Si el sondeo falla, el kubelet mata el contenedor, y el contenedor
+está sujeto a su [política de reinicio](#restart-policy). Si un contenedor no
+tiene un sondeo de inicio, el estado por defecto es `Success`.
+
+Para mayor información sobre como configurar un sondeo liveness,
+readiness o de startup, mire la
+sección [Configurar una sonda Liveness, Readiness y Startup](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).
+
+#### ¿Cuándo debería utilizar un sondeo liveness?
+
+Si el proceso en tu contenedor es capaz de terminar por sí mismo cuando
+encuentra un error o deja de estar sano, no necesitas un sondeo liveness; el
+kubelet automáticamente realizará la acción adecuada de acuerdo con la política
+de reinicio `restartPolicy` del Pod.
+
+Si te gustaría que tu contenedor fuese destruido y reiniciado si falla un
+sondeo, especifica un sondeo liveness, y especifica una `restartPolicy`
+de `Always` o `OnFailure`.
+
+#### ¿Cuándo debería utilizar un sondeo readiness?
+
+Se te gustaría enviar tráfico al Pod solo cuando una sonda sea exitosa,
+especifica un sondeo readiness.
+En este caso,
+el sondeo readiness podría ser el mismo que el liveness,
+pero la existencia del sondeo readines en la especificación significa que el Pod
+iniciará sin recibir ningún tráfico y solo iniciará cuando el sondeo readiness
+sea exitoso.
+
+Si quieres que tu contenedor sea capaz de darse de baja por mantenimiento por sí
+mismo,
+puedes especificar un sondeo de readiness que revisa un endpoint específico de
+readiness que es distinto del sondeo liveness.
+
+Si tu aplicación tiene una dependencia estricta con servicios de trasfondo,
+puedes implementar ambos sondeos de liveness y readiness.
+El sondeo de liveness pasa cuando la aplicación por sí misma está sana, pero el
+sondeo de readiness revisa adicionalmente que cada servicio de trasfondo está
+disponible.
+Esto ayuda a evitar enviar a Pods que solo pueden responder con errores.
+
+Si tu contenedor necesita trabajar cargando grandes datos, ficheros de
+configuración, o migraciones durante el inicio, puedes usar
+un [sondeo de inicio](#when-should-you-use-a-startup-probe).
```

Comment:
Use the same words to have the reference working:

```suggestion
un [sondeo de inicio](#Cuando-deberia-utilizar-un-sondeo-de-inicializacion).
```

or, put the reference in English below as:

```yaml
#### ¿Cuándo debería utilizar un sondeo de inicialización? {#when-should-you-use-a-startup-probe}
```